pkg/cdi/schema.go

```diff
@@ -0,0 +1,48 @@
+/*
+   Copyright © 2021 The CDI Authors
```

Comment:
Nit: Should this date be updated or include a range?

Reply:
True, I updated the date.